experimental/oak_baremetal_channel/src/lib.rs

```diff
@@ -14,11 +14,13 @@
 // limitations under the License.
 //
 
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
 #![feature(never_type)]
 #![allow(rustdoc::private_intra_doc_links)]
 
+#[cfg(feature = "client")]
 pub mod client;
+
 pub mod server;
```

Comment:
You introduce the `server` feature in `Cargo.toml`, above; should this be gated on that feature being enabled, much like the client is?

Reply:
Ah thanks for catching, overlooked the server feature. Removed that feature, imo we don't need it right now. The reason the client code needs feature gating is that it introduces a dependency on the standard library. The server code if not used can just be compiled out.